lms/static/scripts/frontend_apps/components/FilePickerFormFields.js

```diff
@@ -34,9 +35,13 @@ export default function FilePickerFormFields({
   formFields,
   groupSet,
   ltiLaunchURL,
+  hypothesisId,
 }) {
   /** @type {Record<string,string>} */
   const extraParams = groupSet ? { group_set: groupSet } : {};
+  if (hypothesisId) {
```

Comment:
Should we not send the url of the document to canvas anymore? I think we still should send it even if we don't use it.